mu-plugins/blocks/screenshot-preview/src/block.js

```diff
@@ -20,6 +20,7 @@ import ScreenShot from './screenshot';
  * @param {string} props.width       Desired with of the preview element, include unit.
  * @param {number} props.aspectRatio Aspect ratio for the preview element.
  * @param {string} props.queryString Arguments passed to screenshot service.
+ * @param {Object} props.tagProps    Props added to block tag.
```

Comment:
As this object is spread directly onto the tag it would be pretty easy to mistakenly add invalid properties to this. Might be worth being explicit about this, for example the Typescript type would be HTMLAnchorElement. 

Maybe
```suggestion
 * @param {Object} props.anchorTagProps    HTMLAnchorElement attributes to be added to the block anchor tag.
```

Reply:
Yeah, I was hoping in the future we could make the component more flexible and use something like  `as="button"` since not every use case navigates. The reason it's still an `<a>` tag is that theme directory styling makes a mess of buttons and I'm not certain patching all those styles at the moment will be the most effective approach. The theme directory design updates will lend more context to this when we get there.

But good suggestion, I'll make the change since it's a better reflection of how the code currently behaves.